Added some clarity to the initial setup section (#3113)
segmentio/segment-docs: src/connections/destinations/catalog/actions-metronome/index.md

Metronome also enables product-led growth with a consistent source of truth for use and billing. Freely experiment with pricing and packaging and put iteration directly in the hands of your Product team.

> ## Initial Setup

Reviewer: ```suggestion
## Getting Started
```

For consistency with the rest of our integrations docs, we'll leave this as `Getting Started` for now. We're looking at a broader update here though to make things more clear.

1. From the Segment web app, click **Catalog**, then click **Destinations**

> 2. Search for ''Metronome (Actions)'' within the Destinations Catalog and select **Metronome (Actions)**

Reviewer: ```suggestion
2. Search for "Metronome (Actions)" within the Destinations Catalog and select **Metronome (Actions)**
```

Reviewer 2: I think this should be kept as bolded rather than adding ""

3. Click **Configure Metronome (Actions)**.

> 4. Select the source you’d like to connect to and give the destination a name (e.g., Metronome Instance #1).

Reviewer: ```suggestion
4. Select the source you’d like to connect to and give the destination a name (for example, `Metronome Instance #1`).
```
